.github/workflows/update-loci.yaml

```diff
@@ -0,0 +1,57 @@
+name: Update data
+
+on:
+  pull_request:
+    branches: main
+    paths:
+      - "data/**"
+      - "scripts/**"
+      - "workflow/**"
+      - ".github/workflows/**"
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+
+    steps:
+      - name: Debug dump
+        uses: crazy-max/ghaction-dump-context@v2
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Miniconda environment
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          activate-environment: strchive
+          environment-file: scripts/environment.yml
+          python-version: 3.12
+          auto-activate-base: false
+      - run: |
+          conda info
+
+      - name: Check loci and update bed files (short)
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'}}
```

Comment:
Since you've removed the `schedule` trigger for this workflow, these `if` conditions are no longer needed.